beaker/application.py

```diff
@@ -69,9 +70,19 @@ class Application:
     address: Final[Expr] = Global.current_application_address()
     id: Final[Expr] = Global.current_application_id()
 
-    def __init__(self, version: int = MAX_TEAL_VERSION):
+    def __init__(
+        self, teal_version: int = MAX_TEAL_VERSION, version: int | None = None
```

Comment:
We went thru a renaming process with pyteal and other repos to assimilate the version/naming between AVM version and teal/pragma version.

PyTEAL also has a concept of `pragma` here: https://github.com/algorand-devrel/beaker/blob/master/examples/amm/amm.py#L40

Can this change be removed from the pr and filed separately so we can discuss in isolation from the rest of these?

Reply:
Yeah sure, I'll put it into a different PR

Reply:
Moved to #172